gtsfm/utils/ellipsoid.py

```diff
@@ -31,11 +31,15 @@ def get_ortho_axis_alignment_transform(gtsfm_data: GtsfmData) -> Pose3:
 
     # Filter outlier points, Center point cloud, and obtain alignment rotation.
     points_filtered = remove_outlier_points(point_cloud)
-    points_centered, mean = center_point_cloud(points_filtered)
+    points_centered = center_point_cloud(points_filtered)
     wuprightRw = get_alignment_rotation_matrix_from_svd(points_centered)
 
+    # Calculate translation vector based off rotated point cloud.
+    point_cloud_rotated = (wuprightRw @ point_cloud.T).T
```

Comment:
I think `point_cloud @ wuprightRw.T` is cleaner IMHO

Reply:
Agreed